webapp/app/templates/basis/base_step_form.html

```diff
@@ -39,4 +39,50 @@
         $('input[type="date"]').attr('max', '9999-12-31');
     </script>
     <script src="{{ url_for('static', filename='js/jquery.mask.min.js') }}" type=text/javascript></script>
+    <script>
+            $(document).ready(function(){
+                $('.date_input').mask('00.00.0000');
+                $('input[name="unlock_code"]').mask('A', {
+                    translation: {
+                        "A": { pattern: /[\w]/, recursive: true }
+                    }
+                });
+                window.memo_values  = new Array();
+
+                 const sub_fields = document.querySelectorAll('input[name="unlock_code"]');
+                 sub_fields.forEach(input => {
+                     input.addEventListener('input', e => {
+                         console.log("old value" + window.memo_values[input.id]);
+                         console.log("Hallo" + input.value);
+                         const data = input.value;
```

Comment:
You can use `e.target` instead of `input` here. It's a bit cleaner because it doesn't rely on the reference from the closure and makes it easier to extract the function later if we need to.

Reply:
This script was added by mistake (s. below)